deploy/cephfs/kubernetes/csi-cephfsplugin.yaml

```diff
@@ -18,23 +18,23 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet      
       containers:
         - name: driver-registrar
-          image: quay.io/k8scsi/driver-registrar:canary
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
           args:
             - "--v=5"
-            - "--csi-address=$(ADDRESS)"
-            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+            - "--csi-address=/csi/csi.sock"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/csi-cephfsplugin/csi.sock"
```

Comment:
@rootfs, any specific reson for switching from `plugins_registry` to `plugins` ?

I think for CSI 1.0 onwards we have to use `plugins_registry` https://github.com/kubernetes/kubernetes/blob/7f0e04a089125901dce18c7d96507f2b60560e18/pkg/volume/csi/csi_plugin.go#L123
please correct me if am wrong.

Reply:
see https://github.com/kubernetes-csi/node-driver-registrar